test/real_content/real_content_lib.bash

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+source "test/functional/testlib.bash"
```

Comment:
Why did this change from: load "../functional/testlib"?

With source, I can only run the tests from the top level directory

Reply:
I just got that. load is a command in bats and this is a bash script, so we need to use source. The path thing is very inconvenient, you are right about that. Is there any way to fix it? I really don't know.

Reply:
Yaaa, I see that now.... We could get the path to the script:

**SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"**